chcemvediet/tests/__init__.py

```diff
@@ -112,7 +112,7 @@ def _create_neighbourhood(self, **kwargs):
     def _create_obligee(self, **kwargs):
         return self._call_with_defaults(Obligee.objects.create, kwargs, {
                 u'official_name': u'Default Testing Official Name',
-                u'name': u'Default Testing Name {}'.format(self.counter.next()),
+                u'name': u'Default Testing Name {:03d}'.format(self.counter.next()),
```

Comment:
@martinmacko47 Obligees sa usporiadaju abecedne podla mena, mame preto napisany aj test `test_autocomplete_returns_obligees_ordered_by_name`. Je teda potrebne formatovat aspon na 2 nuly.

Reply:
@viliambalaz Nerozumiem, ved test `test_autocomplete_returns_obligees_ordered_by_name` by mal zbehnut bez ohladu na to, ci default nazvy institucii su alebo nie su prefixnute nulami. Kedze volanie pouzite v teste vdaka `?term=aaa` institucie s defaultnymi nazvami vobec nevrati.